workflows/sarek.nf

```diff
@@ -1050,15 +1048,25 @@ workflow SAREK {
     if (!(params.skip_tools && params.skip_tools.split(',').contains('multiqc'))) {
         workflow_summary    = WorkflowSarek.paramsSummaryMultiqc(workflow, summary_params)
         ch_workflow_summary = Channel.value(workflow_summary)
+        methods_description    = WorkflowSarek.methodsDescriptionText(workflow, ch_multiqc_custom_methods_description)
+        ch_methods_description = Channel.value(methods_description)
 
-        ch_multiqc_files =  Channel.empty().mix(ch_version_yaml,
-                                            ch_workflow_summary.collectFile(name: 'workflow_summary_mqc.yaml'),
-                                            ch_reports.collect().ifEmpty([]))
+        ch_multiqc_files = Channel.empty()
+        ch_multiqc_files = ch_multiqc_files.mix(ch_version_yaml)
+        ch_multiqc_files = ch_multiqc_files.mix(ch_workflow_summary.collectFile(name: 'workflow_summary_mqc.yaml'))
+        ch_multiqc_files = ch_multiqc_files.mix(ch_methods_description.collectFile(name: 'methods_description_mqc.yaml'))
+        ch_multiqc_files = ch_multiqc_files.mix(ch_reports.collect().ifEmpty([]))
 
         ch_multiqc_configs = Channel.from(ch_multiqc_config).mix(ch_multiqc_custom_config).ifEmpty([])
 
-        MULTIQC(ch_multiqc_files.collect(), ch_multiqc_configs.collect())
+        MULTIQC (
+            ch_multiqc_files.collect(),
+            ch_multiqc_config.collect().ifEmpty([]),
+            ch_multiqc_custom_config.collect().ifEmpty([]),
+            ch_multiqc_logo.collect().ifEmpty([])
+        )
         multiqc_report = MULTIQC.out.report.toList()
+        ch_versions    = ch_versions.mix(MULTIQC.out.versions)
```

Comment:
hmm ... Is the channel `ch_versions` used after that point in the code?

Reply:
not that I'm aware of

Reply:
hmm ... if it isn't used then why is line 1069 there?

Reply:
it was in TEMPLATE, but I agree, don't make much sense.
I actually have some ideas for that later on, but for now, we can keep it

Reply:
ok